composer.json

```diff
@@ -44,7 +44,7 @@
         "symfony/console": "^6.2",
         "symfony/error-handler": "^6.2",
         "symfony/finder": "^6.2",
-        "symfony/http-foundation": "^6.3",
+        "symfony/http-foundation": "^6.4",
```

Comment:
This dependency in component `composer.json` files also needs to be bumped.

Reply:
I noticed `illuminate/session` doesn't use the `illuminate/cookie` dependency. Am I missing something or is it correct to add the dependency?

Reply:
Sorry, I didn't check it properly. `StartSession` uses the Symfony Cookie. I will bump the version there as well.

Reply:
@fabricecw there's more components that use `symfony/http-foundation`. Can you do a search and bump all of them please?

Reply:
Absolutely. The other components would't necessarily require 6.4. But I guess it's a good idea to use 6.4 everywhere.